SM failure is unrelated

JDK 11 failure is due to `Caused by: java.lang.RuntimeException: java.lang.RuntimeException: java.nio.file.FileSystemException: ...  No space left on device`

--- a/ejb3/src/main/resources/subsystem-templates/ejb3.xml
+++ b/ejb3/src/main/resources/subsystem-templates/ejb3.xml
@@ -45,6 +45,7 @@
        <?IIOP?>
        <default-security-domain value="other"/>
        <default-missing-method-permissions-deny-access value="true" />
+       <statistics enabled="${wildfly.ejb3.statistics-enabled:${wildfly.statistics-enabled:false}}"/>
        <log-system-exceptions value="true" />
    </subsystem>
    <supplement name="default">

--- a/galleon-pack/src/main/resources/feature_groups/ejb3.xml
+++ b/galleon-pack/src/main/resources/feature_groups/ejb3.xml
@@ -9,6 +9,7 @@
         <param name="default-singleton-bean-access-timeout" value="5000"/>
         <param name="default-missing-method-permissions-deny-access" value="true"/>
         <param name="log-system-exceptions" value="true"/>
+        <param name="statistics-enabled" value="${wildfly.ejb3.statistics-enabled:${wildfly.statistics-enabled:false}}"/>
         <feature spec="subsystem.ejb3.cache">
             <param name="cache" value="simple"/>
         </feature>

--- a/galleon-pack/src/main/resources/feature_groups/webservices.xml
+++ b/galleon-pack/src/main/resources/feature_groups/webservices.xml
@@ -2,6 +2,7 @@
 <feature-group-spec name="webservices" xmlns="urn:jboss:galleon:feature-group:1.0">
     <feature spec="subsystem.webservices">
         <param name="wsdl-host" value="${jboss.bind.address:127.0.0.1}"/>
+        <param name="statistics-enabled" value="${wildfly.webservices.statistics-enabled:${wildfly.statistics-enabled:false}}"/>
         <feature spec="subsystem.webservices.client-config">
             <param name="client-config" value="Standard-Client-Config"/>
         </feature>

--- a/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/management/deployments/EjbInvocationStatisticsTestCase.java
+++ b/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/management/deployments/EjbInvocationStatisticsTestCase.java
@@ -106,7 +106,7 @@ public class EjbInvocationStatisticsTestCase {
     private static Boolean getEnableStatistics(final ManagementClient managementClient) throws IOException {
         final ModelNode address = PathAddress.pathAddress(PathElement.pathElement(SUBSYSTEM, SUBSYSTEM_NAME)).toModelNode();
         address.protect();
-        final ModelNode value = readAttribute(managementClient, address, "enable-statistics");
+        final ModelNode value = readAttribute(managementClient, address, "statistics-enabled");
         if (value.isDefined())
             return value.asBoolean();
         return null;
@@ -117,6 +117,7 @@ public class EjbInvocationStatisticsTestCase {
         op.get(ModelDescriptionConstants.OP).set(ModelDescriptionConstants.READ_ATTRIBUTE_OPERATION);
         op.get(ModelDescriptionConstants.OP_ADDR).set(address);
         op.get(ModelDescriptionConstants.NAME).set(attributeName);
+        op.get(ModelDescriptionConstants.RESOLVE_EXPRESSIONS).set(true);
         return execute(managementClient, op);
     }
 

--- a/testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap700/DomainAdjuster700.java
+++ b/testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap700/DomainAdjuster700.java
@@ -25,6 +25,7 @@ package org.jboss.as.test.integration.domain.mixed.eap700;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.EXTENSION;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
 import static org.jboss.as.controller.operations.common.Util.createRemoveOperation;
+import static org.jboss.as.controller.operations.common.Util.getWriteAttributeOperation;
 
 import java.util.ArrayList;
 import java.util.Collection;
@@ -48,6 +49,7 @@ public class DomainAdjuster700 extends DomainAdjuster710 {
 
         list.addAll(removeCoreManagement(profileAddress.append(SUBSYSTEM, "core-management")));
         list.addAll(removeElytron(profileAddress.append(SUBSYSTEM, "elytron")));
+        list.addAll(adjustWS(profileAddress.append(SUBSYSTEM, "webservices")));
         return list;
     }
 
@@ -65,6 +67,13 @@ public class DomainAdjuster700 extends DomainAdjuster710 {
         //core-management subsystem and extension don't exist
         list.add(createRemoveOperation(subsystem));
         list.add(createRemoveOperation(PathAddress.pathAddress(EXTENSION, "org.wildfly.extension.core-management")));
+        return list;
+    }
+
+    private Collection<? extends ModelNode> adjustWS(PathAddress webservices) {
+        final List<ModelNode> list = new ArrayList<>();
+        //expressions are not enabled for statistics-enabled attribute
+        list.add(getWriteAttributeOperation(webservices, "statistics-enabled", false));
         return list;
     }
 }

--- a/webservices/server-integration/src/main/resources/subsystem-templates/webservices.xml
+++ b/webservices/server-integration/src/main/resources/subsystem-templates/webservices.xml
@@ -2,7 +2,7 @@
 <!--  See src/resources/configuration/ReadMe.txt for how the configuration assembly works -->
 <config>
    <extension-module>org.jboss.as.webservices</extension-module>
-   <subsystem xmlns="urn:jboss:domain:webservices:2.0">
+   <subsystem xmlns="urn:jboss:domain:webservices:2.0" statistics-enabled="${wildfly.webservices.statistics-enabled:${wildfly.statistics-enabled:false}}">
        <wsdl-host>${jboss.bind.address:127.0.0.1}</wsdl-host>
        <endpoint-config name="Standard-Endpoint-Config"/>
        <endpoint-config name="Recording-Endpoint-Config">

